packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart

```diff
@@ -267,7 +267,9 @@ class _AliceCallsListPageState extends State<AliceCallsListPage>
             ? context.i18n(AliceTranslationKey.saveSuccessView)
             : null,
         secondButtonAction: () =>
-            OperatingSystem.isAndroid ? OpenFilex.open(result.path) : null,
+            OperatingSystem.isAndroid && result.path != null
```

Comment:
This shouldn't be part of this PR and is already being addressed in https://github.com/jhomlala/alice/pull/229

Reply:
Updated! Do we know when this will be merged?

Reply:
What do you mean by "this"? 😅 This PR here or #229?

Reply:
Ideally both, but I meant this specific PR. Sorry for not being specific. 🙃

Reply:
It will probably have to come after #229 as without it the CI won't be able to deploy it.

Obviously, I can't approve my own PR (#229) so we'll have to wait on @jhomlala to approve it.